src/reducers/versions.tsx

```diff
@@ -463,6 +468,14 @@ export const getVersionInfo = (versions: VersionsState, versionId: number) => {
   return versions.versionInfo[versionId];
 };
 
+export const getPrunedExpandedPaths = (
+  expandedPaths: string[],
+  version: Version,
+) => {
+  const paths = new Set(version.entries.map((e) => e.path));
+  return expandedPaths.filter((e) => paths.has(e));
```

Comment:
Since `e` is not an entry (as it is above), I suggest picking a different variable like `p` (for path) or something.